### PR TITLE
multiple domain support

### PR DESCRIPTION
**Pull request by Lec1k**
#257

### CODE REVIEW

File: lib/howitzer.rb [hunk 1]
```diff
@@ -37,12 +37,13 @@ def mailgun_idle_timeout
   # @example returns url without auth
   #  app_uri.origin
 
-  def self.app_uri
+  def self.app_uri(name = nil)
```
Comment (romikoops) [inline lib/howitzer.rb]: unit tests are missing

Comment (romikoops) [reply lib/howitzer.rb]: If user forgot to create the settings, howitzer needs to show Warning: You used host, but forgot to define settings, please add following settings:...

Comment (romikoops) [reply lib/howitzer.rb]: Change log should be update

Comment (romikoops) [reply lib/howitzer.rb]: code documentation should be extended

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ### New Features
 - Cucumber rake tasks minor updates
+- Add multiple domain support
```
Comment (romikoops) [inline CHANGELOG.md]: We have issue for it. It means, we need to provide link to original issue

File: lib/howitzer.rb [hunk 3]
```diff
@@ -36,14 +36,25 @@ def mailgun_idle_timeout
   #  app_uri.site
   # @example returns url without auth
   #  app_uri.origin
+  # @example returns url for custom host(should be specified in config)
```
Comment (romikoops) [inline lib/howitzer.rb]:
```ruby
# @example returns url for custom host
# app_uri(:example).site
```

File: lib/howitzer.rb [hunk 4]
```diff
@@ -36,14 +36,25 @@ def mailgun_idle_timeout
...
-      user: Howitzer.app_base_auth_login,
-      password: Howitzer.app_base_auth_pass,
-      host: Howitzer.app_host,
-      scheme: Howitzer.app_protocol || 'http'
+      user: Howitzer.public_send("#{prefix}app_base_auth_login"),
+      password: Howitzer.public_send("#{prefix}app_base_auth_pass"),
+      host: Howitzer.public_send("#{prefix}app_host"),
+      scheme: Howitzer.public_send("#{prefix}app_protocol") || 'http'
     )
+  rescue NoMethodError
```
Comment (romikoops) [inline lib/howitzer.rb]: Looks like settings are validating in SexySettings automatically. Do we really need the message?

Comment (Lec1k) [reply lib/howitzer.rb]: If setting is not present it throws exception like this:
```
undefined method `howitzer_app_base_auth_login' for Howitzer:Module (NoMethodError)
```
With this rescue message becomes more clear:
```
config is missing for custom host howitzer, please add to config:
howitzer_app_base_auth_login
howitzer_app_base_auth_pass
howitzer_app_host
howitzer_app_protocol (NoMethodError)
```

**Pull request merged**